### DISCUSSION TITLE
Unable To Create OAuth Clients (HTML 422)

### DISCUSSION THREAD

**Discussion by shenhong96** [firefly-iii/firefly-iii#6504] [General]
Not sure whether is this a bug or i had done something wrong.

I deployed and configured the app and so far so good. Problem being unable to create Oauth Client so i can use an IOS app "Abacus" to connect to my firefly iii backend.

When i go to the UI -> "Create New Client", nothing happened. 
only 1 line relevant in the logs

 firefly-app_1  | 192.168.x.x - - [05/Oct/2022:09:33:13 +0800] "POST /oauth/clients HTTP/1.1" 422 1922 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/16.0 Safari/605.1.15"

<img width="599" alt="image" src="https://user-images.githubusercontent.com/58160889/193966126-030b9509-0234-46b1-b819-a8b0d4435e6b.png">

<img width="442" alt="image" src="https://user-images.githubusercontent.com/58160889/193966187-c7abb2df-dcfc-417a-a82d-f967e1c1b2d8.png">

I installed using docker-compose. Tried to create through official demo site and it works.

Debug information generated at 2022-10-05 10:32:03 Asia/Kuala_Lumpur for Firefly III version **5.6.5**.

| Scope | Version |
| --- | --- |
| Firefly III | 5.6.5 |
| Firefly III API | 1.5.4 |
| PHP | 8.0.12 |
| Host | Linux |

| System info | Value |
| --- | --- |
| Installation ID | 8f4263e1-2625-4159-8811-889bad4010e1 |
| Using docker? | true |
| Telemetry | false |
| Layout | v1 |
| System TZ | Asia/Kuala_Lumpur |
| Browser TZ | Asia/Kuala_Lumpur |
| App environment | local |
| App debug mode | false |
| App cache driver | file |
| App logging | debug, stack |
| Display errors | Off |
| Error reporting | ALL errors |
| Interface | apache2handler |
| Default language | en_US |
| Default locale | equal |
| BCscale | 24 |
| DB drivers | mysql, pgsql, sqlite |
| Current driver | mysql |
| DB version | 18 (exp. 18) |
| Login provider | eloquent  |
| Trusted proxies (.env) | ** |

| User info | Value |
| --- | --- |
| Session start | 2022-10-01 00:00:00 |
| Session end | 2022-10-31 23:59:59 |
| Session first | 2022-10-04 21:41:23 |
| User ID | 1 |
| User language | en_US |
| User locale | en_US |
| Attempt at "en_US.utf8" | 'en_US.utf8' |
| Attempt at "en_US.UTF-8" | 'en_US.UTF-8' |
| User agent | Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/16.0 Safari/605.1.15 |

**Comment by JC5** [comment]
422, it may not recognize the prefix. But upgrade to 5.7.12 first, that may already fix the issue.

**Comment by shenhong96** [comment]
Goodness. So sry, i din't realize I used an old dangling image with so many version gap because this was my second try to set this up.

Thank you so much for your previous help and prompt reply.  

Pulling the latest version solves the problem.
